intrahospital_api/loader.py

```diff
@@ -367,40 +130,31 @@ def async_load_patient(patient_id, patient_load_id):
         raise
 
 
+@timing
+def load_demographics(hospital_number):
+    started = timezone.now()
+    api = get_api()
+    try:
+        result = api.demographics(hospital_number)
+    except:
+        stopped = timezone.now()
+        logger.info("demographics load failed in {}".format(
```

Comment:
Wouldn't you want to know demographics load for `hospital_number` ?

Reply:
There is not necessarily a demographics load. This is the function that's called by the add patient pathway.